backend/src/controllers/user.js

```diff
@@ -0,0 +1,49 @@
+const userService = require('../services/user');
+
+async function buyStock(req, res) {
+  const {
+    stockSymbol, sharesBought, price,
+  } = req.query;
+
+  const { id } = req.query;
```

Comment:
Shouldn't this stuff be passed in the request body rather than as query params?

Reply:
@kyzh0 its okay, this will later in req.user, when we validate the token

Reply:
@YungiKim1999 should probably not pass the id in params, or body and change it to user

Reply:
But stockSymbol, shareBought, price should still be passed in the body right?

Reply:
yup